### PR TITLE
Test coverage for active active published events after replay

### PR DESCRIPTION
**Pull request by johanandren**

### CODE REVIEW

File: akka-persistence-typed-tests/src/test/scala/akka/persistence/typed/ActiveActiveEventPublishingSpec.scala [hunk 1]
```diff
@@ -145,6 +157,38 @@ class ActiveActiveEventPublishingSpec
       probe.expectMessage(Set("one", "two", "three"))
     }
 
+    "handle published events after replay" in {
```
Comment (chbatey) [inline akka-persistence-typed-tests/src/test/scala/akka/persistence/typed/ActiveActiveEventPublishingSpec.scala:160]: i think we should also handle a filtered message

Comment (chbatey) [reply akka-persistence-typed-tests/src/test/scala/akka/persistence/typed/ActiveActiveEventPublishingSpec.scala:160]: as i think that is broken as we don't start with the correct seenPerReplica

Comment (johanandren) [reply akka-persistence-typed-tests/src/test/scala/akka/persistence/typed/ActiveActiveEventPublishingSpec.scala:160]: Or a published before stopping would trigger the same I guess.

**Comment by johanandren** [comment]
Snuck in (at least seqnr tracking part of) #29256 here as well

**Review by chbatey — APPROVED** [review]

**Pull request merged**